### DISCUSSION TITLE
Why a package trigger another's download even when I have  built-in in the other's recipe?

### DISCUSSION THREAD

**Discussion by WorldsEndless** [raxod502/straight.el#1162] [Q&A]
Why does elfeed-org always clone a copy of org even when my org recipe has `straight (:type built-in)`? After an annoying bit of bisection I found that without elfeed-org, the built-in version is respected and a straight repo is NOT cloned. With elfeed-org, however, I get an org repo and m-x org-version telling me "mixed version." May any package trigger a parent package build, or is there something peculiar about elfeed-org? Is there anything I can do to get elfeed-org to stop triggering the org repo?

**Comment by progfolio** [comment]
Are you able to reproduce the issue via `straight-bug-report`?
That would help narrow down the issue. Thanks.

**Comment by raxod502** [comment]
As @progfolio said. But the most likely cause would be you need to make sure the org recipe is loaded first, else it would get cloned as a dependency of elfeed before straight.el knows it is supposed to be built-in.

**Comment by paniabhisek** [comment]
This is happening for me as well with [org-appear](https://github.com/awth13/org-appear).

The following does not work.

```
(use-package org-appear
  :after org
  :straight t)
```

If I create a folder `org/lisp` inside `straight/repos`, It bypasses the cloning. I do not know a cleaner way through code

**Reply by raxod502** [reply]
Why not `(straight-use-package '(org :type built-in))` before loading `org-appear`?
